README.md

```diff
@@ -328,6 +328,7 @@ If data is being replicated to HDFS then Circus Train will use DistCp to copy th
             skip-crc: false
             ssl-configuration-file: /foo/bar/ssl-config
             copier-factory-class: com.hotels.bdp.circustrain.distcpcopier.DistCpCopier
+            client-manipulation-factory-class: com.hotels.bdp.circustrain.distcpcopier.HdfsDataManipulationClientFactory
```

Comment:
Having looked through this I'm not sure whether we need to expose the term "client". Could we just call it a Data Manipulator? Feels more generic to me. Then this property would become `data-manipulator-factory-class`. Obviously impacts on the rest of the doc so I won't comment again on this below. What do others think?

Reply:
Yeah that works for me. Think the name came from starting out with copying from s3->s3 and needing an s3client to "manipulate the data". Happy to change to this, it reads nicer.